asyncscope.md

```diff
@@ -126,19 +126,18 @@ This will prevent local reasoning, thus will make the program harder to understa
 To properly structure our concurrency, we need an abstraction that ensures that all the work being started has a proper lifetime guarantee.
 This is the goal of `async_scope`.
 
-## P2300 might be blocked by the lack of `async_scope`
+## `async_scope` may increase consensus for P2300 
 
-Although [@P2300R4] is generally considered a strong improvement on concurrency in C++, various people voted against introducing this into the C++ standard.
-The main reason for this negative vote is the absence of a feature like `async_scope`.
+Although [@P2300R5] is generally considered a strong improvement on concurrency in C++, various people voted against introducing this into the C++ standard.
 
-This paper wants to fix this issue, and ensure that [@P2300R4] is not being blocked.
+This paper is intended to increase consensus for [@P2300R5].
 
 Examples of use
 ===============
 
 ## Spawning work from within a task
 
-Using a global `async_scope` in combination with a `system_context` from [@P2079R2] to spawn work from within a task and join it later:
+Use a `async_scope` in combination with a `system_context` from [@P2079R2] to spawn work from within a task and join it later:
```

Comment:
The block in the code below looks like just a random block scope. I think the idea becomes clearer if it is a function definition, so as to illustrate that the system_context is elsewhere. In the Qt<->libunifex integration prototype, it's fetched via QCoreApplication::instance()->thread() when talking to the main event loop.